src/blt_mad/helpers.go

```diff
@@ -171,20 +171,25 @@ func GetValuesLargerThanPercentile(numbers []float64, percentile float64) []floa
 	return largerValues
 }
 
-func SaveArrayToFile(fileName string, arr []float64) error {
-	file, err := os.Create(fileName)
+func AppendFloat64ArrayToTxt(fileName string, floatArray []float64) error {
+	// Open the file in append mode, create if it doesn't exist
+	file, err := os.OpenFile(fileName, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
 	if err != nil {
-		return err
+		fmt.Println("Error opening file:", err)
+		return nil
```

Comment:
Return the error again.

Reply:
Whoops - If this code is commented do you need this?

Disregard previous message.